pymeasure/instruments/texio/texioPSW360L30.py

```diff
@@ -0,0 +1,54 @@
+from pymeasure.instruments import Instrument
+from pymeasure.instruments.keithley.keithley2260B import Keithley2260B
+
+import time
+import logging
+
+log = logging.getLogger(__name__)
+log.addHandler(logging.NullHandler())
+
+
+class TexioPSW360L30(Keithley2260B):
+    r""" Represents the TEXIO PSW-360L30 Power Supply (minimal implementation)
+    and provides a high-level interface for interacting with the instrument.
+
+    For a connection through tcpip, the device only accepts
+    connections at port 2268, which cannot be configured otherwise.
+    example connection string: 'TCPIP::xxx.xxx.xxx.xxx::2268::SOCKET'
+
+    For a connection through USB on Linux, the kernel is going to create
+    a /dev/ttyACMX device automatically. The serial connection properties are
+    fixed at 9600–8-N-1.
+
+    The read termination for this interface is Line-Feed \n.
+
+    This driver was based off the Keithley2260B one.
+
+    .. code-block:: python
+
+        source = TexioPSW360L30("TCPIP::xxx.xxx.xxx.xxx::2268::SOCKET")
+        source.voltage = 1
+        print(source.voltage)
+        print(source.current)
+        print(source.power)
+        print(source.applied)
+    """
+
+    def __init__(self, adapter, read_termination="\n", **kwargs):
```

Comment:
According to your docstring, the read termination is fix "\n": why do you have such a parameter?

If you want to allow the user to change the termination, for some edge cases or subclassing, but not for normal use, I recommend to do it via the kwargs: remove it from the init parameters and add as the first line `kwargs.setdefault('read_termination', "\n").
You would have to remove the parameter from your call to Instrument.init as well.

What this code does, is to inject the your default to the kwargs, if such a key is not present, otherwise it keeps the user's choice.

Reply:
You are right. The read termination is fixed so there is no point in letting the user fiddle with it. I removed the parameter altogether.